### DISCUSSION TITLE
Configuring the red "SEB LOCKED" screen

### DISCUSSION THREAD

**Discussion by taylomik** [SafeExamBrowser/seb-win-refactoring#553] [General]
Hi, Does anyone know if there's a way to configure, alter or disable the red "SEB LOCKED" screen when a PC sleeps? The PCs at our Uni have recently had their idle time drastically reduced and we're now frequently seeing lots of cases of exams being disrupted by this outcome.

We have found nothing in the config files/settings that allows this. Would it be possible to make a change like this without compromising the security of the exam?

Many thanks.

**Comment by danschlet** [comment]
Ideally SEB would distinguish if a PC was just sleeping or if a user account switch was attempted (that's the case in SEB for macOS, but don't know if that's possible in Windows at all). Also I thought the SEB Windows version would also attempt to prevent idle sleep while it is running. But maybe that can be overruled by some deployed system policies (or Windows doesn't allow to prevent idle sleep at all anymore). That sounds weird though, your PC also shouldn't got to sleep while you're watching some movie or streaming(?). @dbuechel can look at this when he's back in February. 

To your question, it's not possible to disable the lock screen, as this would in fact compromise security of the exam. I'm not sure if the lock screen is also used when you disable the setting [Ignore SEB Service](https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane). Then the options from the screen displayed with the ctrl-alt-del shortcut are not available. Maybe it's worth trying that out.

**Reply by taylomik** [reply]
Many thanks for the info - we'll take a look at your suggestion.

Regarding your speculation about why it's sleeping when being used, it's at events such as Maths exams where students are working on paper for significant lengths of time, then entering their answer into a moodle quiz via SEB.

**Reply by dbuechel** [reply]
SEB indeed should prevent sleep and standby. However, as @danschlet already correctly mentioned, these settings may be overridden e.g. by system-wide configuration policies or other deployment and configuration setups for managed devices.

**Reply by dbuechel** [reply]
See https://github.com/SafeExamBrowser/seb-win-refactoring/blob/master/SafeExamBrowser.WindowsApi/NativeMethods.cs#L281
